examples/simple-service-poc/simple_service.py

```diff
@@ -58,20 +59,27 @@ async def get_payload():
         )
 
     async def start(self):
-        # handler responsible for starting the service
+        """handler responsible for starting the service."""
+
+        # perform the initialization of the Service
+        # (which includes sending the network details within the `deploy` command)
         async for script in super().start():
             yield script
-        self._ctx.run(self.SIMPLE_SERVICE_CTL, "--start")
-        yield self._ctx.commit()
+
+        # start the service
+        s = self._ctx.new_script()
```

Comment:
This is a ~~ duplicate of a previous comment, but:

We have `async for script` 3 lines above.
Here we have `s` for a variable that is essentially the same thing as the `script`.
This is confusing.